### ISSUE TITLE
Latest version (5.0.221116) breaks syntax highlighting

### ISSUE THREAD

**Issue by developerdavi** [microsoft/vscode-typescript-next#43]
### The issue
Syntax highlighting stopped working for TypeScript files after last update.

v5.0.20221115:
![image](https://user-images.githubusercontent.com/31714350/202331842-efdd5bab-85c1-4e94-bea4-107bfbe74c0f.png)

v5.0.20221116:
![image](https://user-images.githubusercontent.com/31714350/202331923-e2308b19-3f04-4742-9148-f7c8f256eb0f.png)

### Logs
```
2022-11-16 22:33:25.758 [error] Unable to load and parse grammar for scope source.tsx from file:///Users/devdavi/.vscode/extensions/ms-vscode.vscode-typescript-next-5.0.20221116/syntaxes/TypeScriptReact.tmLanguage.json {}
```

### Steps to reproduce

- Update to latest version (or let vscode update it automatically as happened to me)
- Open a .ts (TypeScript) or .tsx (TypeScript React) file
- You will probably see that syntax highlighting is not working anymore

**Comment by Just1dream** [comment]
same to me

**Comment by bdabin** [comment]
Me too !!!

**Comment by Just1dream** [comment]
I temporarily disabled JavaScript and the TypeScript Nightly plugin to fix the problem

**Comment by developerdavi** [comment]
Manually downgrading to the yesterday version fixes the issue as well.

**Comment by GersonRocha9** [comment]
Same problem to me

**Comment by wujihua118** [comment]
same problem to me, disable or downgrade first

**Comment by developerdavi** [comment]
As seen in last commit, the typescript version was upgraded as well and that can be actually an issue with that new version of the typescript package.

![image](https://user-images.githubusercontent.com/31714350/202334715-dcbfbfda-a9bb-456c-bb8d-9bb4ccf0c042.png)

On the [typescript repo](https://github.com/microsoft/TypeScript) last commits [they changed some things](https://github.com/microsoft/TypeScript/commit/f4ddc1a09bf55a911a2fc5aa1e5209509583ef3d) that potentially caused that issue, as the commit stopped pipelines to succeed.

As it's a nightly build, sounds very reasonable to just downgrade to the previous version and wait the core team to fix that.

**Comment by fahmij8-tkp** [comment]
happened to me as well

<img width="305" alt="image" src="https://user-images.githubusercontent.com/106066666/202349098-535181d0-bf1f-4025-b525-aea3505e282b.png">

**Comment by soumyajyotibiswas** [comment]
Same here. Using `v5.0.20221115` for now

**Comment by IdrisAkintobi** [comment]
Downgrading to the earlier version also fixed mine. This actually took a few minutes out of my life. I couldn't afford to code without the syntax highlighting 😂. Respect to the developers who coded in plain text.

**Comment by J-Dany** [comment]
Same here. Using version `5.0.20221116`

**Comment by yangfan-coder** [comment]
> 步骤 1. 遇到麻烦 <img alt="图片" width="723" src="https://user-images.githubusercontent.com/55613446/202399407-5079c69e-4394-4f44-9d42-d2595610eba3.png">
> 
> 步骤 2. 检查“JavaScript 和 TypeScript Nightly”的版本 <img alt="图片" width="768" src="https://user-images.githubusercontent.com/55613446/202398395-781c7bdc-5fbf-4b68-bb62-58cef3042bac.png">
> 
> 步骤 3. 更改版本 v5.0.20221115 <img alt="图片" width="619" src="https://user-images.githubusercontent.com/55613446/202399753-61d019fb-711f-47bc-ab7d-cd760a74aee0.png"> <img alt="图片" width="655" src="https://user-images.githubusercontent.com/55613446/202399877-070861c8-f45c-4e2c-b265-7ad1ea3fcccf.png">
> 
> 第 4 步。仍然有麻烦。不要恐慌。只需重新加载 command + shift + p -> Developer:Reload Window <img alt="图片" width="596" src="https://user-images.githubusercontent.com/55613446/202400043-4be63912-fd7d-4646-b976-5d63dc1080e2.png"> <img alt="图片" width="646" src="https://user-images.githubusercontent.com/55613446/202400290-d67bff33-ba4e-44b5-922c-e1c6bdbf15da.png">
> 
> 步骤 5. 求解！ <img alt="图片" width="704" src="https://user-images.githubusercontent.com/55613446/202400555-c914d2a5-8692-4df0-93e8-8ddee9dc0bfe.png">
===========
Thank him for solving my problem！！

**Comment by mjbvz** [comment]
This is fixed with v5.0.202211170

Sorry for the trouble. The root cause was an upstream part of our build pipelines corrupting syntax files
